paddle/optimizer/CMakeLists.txt

```diff
@@ -9,11 +7,6 @@ set(OPITMIZER_SRCS
     sgd_optimizer.cc
   )
 
-add_library(paddle_optimizer STATIC ${OPITMIZER_SRCS})
-add_dependencies(paddle_optimizer paddle_proto ${external_project_dependencies})
-
-
-if(WITH_TESTING)
-  add_simple_unittest(serialization_test)
-  add_simple_unittest(parameter_optimizer_test)
-endif()
+cc_library(paddle_optimizer SRCS ${OPITMIZER_SRCS} DEPS paddle_proto gtest glog)
```

Comment:
TeamCity complains

```
[14:33:58]	CMake Error at cmake/generic.cmake:189 (add_dependencies):
[14:33:58]	The dependency target "gtest" of target "paddle_optimizer" does not exist.
[14:33:58]	Call Stack (most recent call first):
[14:33:58]	paddle/optimizer/CMakeLists.txt:10 (cc_library)
```

It seems that `paddle_optimizer` doesn't depend on `gtest`.

Reply:
Yes, it is. Thanks!